src/main/java/org/iac2/common/model/InstanceModel.java

```diff
@@ -17,6 +17,10 @@ public InstanceModel(DeploymentModel deploymentModel) {
         this.deploymentModel = deploymentModel;
         this.properties = new HashMap<>();
     }
+
```

Comment:
I'm not sure why we need this? To force EDMM to reload changes?

Reply:
Yes. The `EDMM` class works on the `EntityGraph` instance inside the `DeploymentModel`. However, the `DeploymentModel` only reads information from the graph while it is being created.